features/email_notification.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Email notification
+  As a Pension Wise customer who has had a phone appointment
```

Comment:
I think this scenario could be cleaned up a bit...

The subject is "as a Pension Wise customer" but the other steps are referring to the customer as "them" and "I" seems to be the pension wise administrator.